linuxNew/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec

```diff
@@ -0,0 +1,200 @@
+%global upstream_version 8u302-b08
+# Only [A-Za-z0-9.] allowed in version:
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
+# also not very intuitive:
+#  $ rpmdev-vercmp 8.0.302.0.1___8 8.0.302.0.0+8
+#  8.0.302.0.0___8 == 8.0.302.0.0+8
+%global spec_version 8.0.302.0.0.8
+%global spec_release 1
+%global priority 1111
+
+%global source_url_base https://github.com/adoptium/temurin8-binaries/releases/download
+%global upstream_version_no_dash %(echo %{upstream_version} | sed 's/-//g')
+%global java_provides openjdk
+
+# Map architecture to the expected value in the download URL; Allow for a
+# pre-defined value of vers_arch and use that if it's defined
+#  x86_64 => x64
+#  i668 = x86
+%if %{!?vers_arch:1}0
+%ifarch x86_64
+%global vers_arch x64
+%else
+%ifarch %{ix86}
+%global vers_arch x86
+%else
+# Catch-all, use _arch value
+%global vers_arch %{_arch}
+%endif
+%endif
+%endif
+
+Name:        temurin-8-jdk
+Version:     %{spec_version}
+Release:     %{spec_release}
+Summary:     Eclipse Temurin 8 JDK
+
+Group:       java
+License:     GPLv2 with exceptions
+Vendor:      Eclipse Foundation
+URL:         https://projects.eclipse.org/projects/adoptium
+Packager:    Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org>
+
+AutoReqProv: no
+Prefix: /usr/lib/jvm/%{name}
+
+BuildRequires:  tar
+BuildRequires:  wget
+
+Requires: /bin/sh
+Requires: /usr/sbin/alternatives
+Requires: ca-certificates
+Requires: dejavu-sans-fonts
+Requires: libX11%{?_isa}
+Requires: libXext%{?_isa}
+Requires: libXi%{?_isa}
+Requires: libXrender%{?_isa}
+Requires: libXtst%{?_isa}
+Requires: alsa-lib%{?_isa}
+Requires: glibc%{?_isa}
+Requires: zlib%{?_isa}
+Requires: fontconfig%{?_isa}
+Requires: freetype%{?_isa}
+
```

Comment:
The package is currently x86_64 only, but once ARMv7 is added, a libatomic dependency is required. https://rpm.org/user_doc/arch_dependencies.html describes how to do it:

```
%ifarch %ix86
Requires: %{name}.(i?86|athlon|geode)
%endif
```